src/pages/Login/index.tsx

```diff
@@ -119,7 +120,41 @@ const LoginPage = ({ type }: ILoginPage): JSX.Element => {
       content = (<BusinessForm onSubmit={onSubmit}/>);
       break;
     case 'login':
-      content = null;
+      onSubmit = async (data: LoginFormData) => {
+        const businessId = 1234;
```

Comment:
This value is hard coded. You should add an input for the user to add their businessId. This value was created along with the profile and root account in the business creation process. 
By making it an input, we will be able to allow more than just a single business 😀